handbook/people-ops/travel.md

```diff
@@ -1,98 +1,166 @@
 # Company sponsored events (team and company-wide)
 
 
-## Phase I: Teammate Travel Meetup Guidelines
+# Phase II: Teammate Travel Meetup Guidelines
 
-As of June 16th 2021, we will resume reimbursing travel per our [team travel budgets](https://about.sourcegraph.com/handbook/people-ops/travel#team-travel-budgets) for travel and in-person events that are permitted by law. We want this to be gradual, and all events must be small and completely optional. 
+As of June 16th 2021, we have resumed reimbursing travel per our  [team travel budgets](https://about.sourcegraph.com/handbook/people-ops/travel#team-travel-budgets)  for travel and in-person events that are permitted by law. We want this to be gradual, and all events must be completely optional.
 
-As an all-remote company, we want everyone to feel included and connected, regardless of where they're located. Choosing to travel or meet is an incredibly personal decision, and we support team members who may not be interested in meeting up at this time.
+As an all-remote company, we want everyone to feel included and connected, regardless of where they’re located. Choosing to travel or meet is an incredibly personal decision, and we support team members who may not be interested in meeting up at this time.
 
+## Guidelines
 
-## **Guidelines:**
-
-- To avoid the perception that these events are mandatory, managers should not organize events for their teams. 
-   	- Managers can attend events and organize events that don't involve their direct reports.
-
--   Events should be outdoors rather than indoors.
+ -  All individual teammates can continue to plan and coordinate events.    
+ - Managers may now plan meetups
+	- Managers must actively gauge the level of comfort within their teams for both the activities you’ll be engaging in, and how you plan on meeting up. *Be inclusive of different interests in the group.*
+	- Managers must be aware that not everyone on your team may be able to join your meetup. If you will be discussing work related things that involve members of the team who are not physically present, you must find a way to include them remotely. *Accessibility to conversations is important, and all teammates must feel included, whether or not they are joining in person.*
+ 
+ - All meetups will continue to remain completely optional for all attendees.
     
--   Keep gatherings small. Use your best judgement and defer to local jurisdictions in determining the number of attendees.
+-   Use your best judgement and defer to local jurisdictions in determining the number of attendees.
     
 -   Follow local laws, customs, and safety regulations during every step of the meetup process.
-
-- Assume people don't want to shake hands, hug, high-five, etc. 
     
--   Don't ask about other people's private health choices or condition attendance on those factors (*such as vaccination status*).
+-   Assume people don’t want to shake hands, hug, high-five, etc.
```

Comment:
I really like this one. (I'm resisting the urge to suggest a Sourcegraph specific greeting dance.)